pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -1066,3 +1069,59 @@ type PuppetDBSDConfig struct {
 	// +optional
 	EnableHTTP2 *bool `json:"enableHTTP2,omitempty"`
 }
+
+type LightSailSDConfig struct {
+	// The AWS region
+	// TODO: seems like an empty string is valid for the Region, should we make it Required
+	// and let it be, or leave it optional? considering that Prometheus does not
+	// check complain if it is nil.
```

Comment:
if it's optional then you want a string pointer but ensure that the string length is >= 1 if not nil.

Reply:
Added `MinLength=1`